doc/workspace-capabilities.md

```diff
@@ -0,0 +1,101 @@
+# Default DevWorkspace cluster permissions
+
+This document outlines the permissions that are provisioned for the DevWorkspace service account by default. The role attached to the workspace ServiceAccount is defined in [rbac.go](../pkg/provision/workspace/rbac.go). As a regular Kubernetes role definition, this is
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: workspace
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - create
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apps
+  - extensions
+  resources:
+  - deployments
+  - replicasets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resourceNames:
+  - workspace-credentials-secret
+  resources:
+  - secrets
+  verbs:
+  - get
+  - create
+  - patch
+  - delete
+- apiGroups:
+  - workspace.devfile.io
+  resources:
+  - devworkspaces
+  verbs:
+  - get
+  - watch
+  - list
+  - patch
+  - update
+- apiGroups:
+  - controller.devfile.io
+  resources:
+  - devworkspaceroutings
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - workspace.devfile.io
+  resources:
+  - devworkspacetemplates
+  verbs:
+  - get
+  - create
+  - patch
+  - update
+  - delete
+  - list
+  - watch
+```
+
+Additional permissions can be bound to the DevWorkspace ServiceAccount as follows:
+
+1. Find the *DevWorkspace ID* for the DevWorksapce in question. This is available on the `.status.devworkspaceId` field in the object, which can be obtained using `jq`:
```

Comment:
```suggestion
1. Find the *DevWorkspace ID* for the DevWorkspace in question. This is available on the `.status.devworkspaceId` field in the object, which can be obtained using `jq`:
```

Reply:
I make this typo 80% of the time. Thanks.